tests/test_ble_interface_advanced.py

```
@@ -552,35 +585,41 @@ def __init__(self):  # pylint: disable=super-init-not-called
             self._eventLoop = None
             self._eventThread = None
 
-        def _delegate_to_bleak(self, method_name: str, *args, **kwargs):
+        def _delegate_to_bleak(
+            self, method_name: str, *args: object, **kwargs: object
+        ) -> Any:
```

Ruff (0.15.4)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_delegate_to_bleak`

(ANN401)